modules/social_features/social_group/src/Plugin/Field/FieldWidget/SocialGroupSelectorWidget.php

```diff
@@ -232,8 +257,11 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
     }
 
     $change_group_node = $sg_settings->get('allow_group_selection_in_node');
-    /** @var \Drupal\Core\Entity\EntityInterface $entity */
-    $entity = $form_state->getFormObject()->getEntity();
+
+    /** @var \Drupal\Core\Entity\EntityFormInterface $form_object */
```

Comment:
We should assert if the form object is returned, before getting entity.

Reply:
Since it's part of a field widget which can be used only for entity form so I'd say that it's guaranteed that the form object will be returned in all cases.